test/dlc_tests/benchmark/sagemaker/mxnet/training/test_performance_mxnet_sm_training.py

```diff
@@ -0,0 +1,65 @@
+import random
+import os
+import re
+import time
+
+import pytest
+
+from invoke.context import Context
+
+from test.test_utils import BENCHMARK_RESULTS_S3_BUCKET
+
+
+# This test can also be performed for 1 node, but it takes a very long time, and CodeBuild job may expire before the
+# test ends.
+@pytest.mark.parametrize("num_nodes", [4], indirect=True)
+def test_mxnet_sagemaker_training_performance(mxnet_training, num_nodes, region, gpu_only, py3_only):
+    """
+    Run MX sagemaker training performance test
+
+    Additonal context: Setup for this function is performed by 'setup_sm_benchmark_mx_train_env' -- this installs
+    some prerequisite packages, pulls required script, and creates a virtualenv called sm_benchmark_venv.
+
+    TODO: Refactor the above setup function to be more obviously connected to this function,
+    TODO: and install requirements via a requirements.txt file
+
+    :param mxnet_training: ECR image URI
+    :param num_nodes: Number of nodes to run on
+    :param region: AWS region
+    """
+    framework_version = re.search(r"1(\.\d+){2}", mxnet_training).group()
```

Comment:
Please change the regular expression here to something like `r"\d+(\.\d+){2}"` instead, because there is no need to restrict this function to only MXNet 1.x.

Reply:
Makes sense.